app/extensions.js

```diff
@@ -449,8 +444,26 @@ module.exports.init = () => {
   }
 
   let loadExtension = (extensionId, extensionPath, manifest = {}, manifestLocation = 'unpacked') => {
-    if (extensionId === config.PDFJSExtensionId) {
+    const isPDF = extensionId === config.PDFJSExtensionId
+    if (isPDF) {
       manifestLocation = 'component'
+      // Override the manifest. TODO: Update manifest in pdf.js itself after enough
+      // clients have updated to Brave 0.20.x+
+      manifest = {
+        name: 'PDF Viewer',
+        manifest_version: 2,
+        version: '1.9.457',
+        description: 'Uses HTML5 to display PDF files directly in the browser.',
```

Comment:
By having this here, do we lose any translation that was being done? Should this be loading using `locale.translation()`?

Reply:
@bsclifton I don't think translation would be lost. It was also a literal string in the old manifest: https://github.com/brave/pdf.js/blob/master/extensions/chromium/manifest.json#L5. AFAIK passing a manifest object to the extension-loading function is the same as reading a manifest.json file, so if it was translated before, it would be translated now.